R/pkg/R/functions.R

```diff
@@ -426,7 +426,7 @@ setMethod("covar_pop", signature(col1 = "characterOrColumn", col2 = "characterOr
 
 #' cos
 #'
-#' Computes the cosine of the given value.
+#' Computes the cosine of the given value. Units in radians.
```

Comment:
Yes, if that one is merged first, we should resolve the conflict here. If this one gets merged first, that one should resolve the conflict. Describing the same information across APIs looks reasonable to me.